Examples/MAX78000/CNN/faceid_evkit/project.mk

```diff
@@ -31,9 +33,10 @@ endif
 # Place build files specific to FTHR_RevA here.
 ifeq "$(BOARD)" "FTHR_RevA"
 # Only Enable if 2.4" TFT is connected to Feather
-#PROJ_CFLAGS+=-DTFT_ENABLE
+PROJ_CFLAGS+=-DTFT_ENABLE
```

Comment:
Please confirm this change is intentional.

Reply:
Thanks, reverted in https://github.com/Analog-Devices-MSDK/msdk/pull/792/commits/4c7454d5bb69854330aed77f6255a9423ee3addc